src/table/cells/handle-cell-focus.js

```diff
@@ -1,25 +1,43 @@
-export const focusCell = (rowElements, nextCellCoord) => {
-  const nextCell = rowElements[nextCellCoord[0]].getElementsByClassName('sn-table-cell')[nextCellCoord[1]];
-  nextCell.focus();
-  nextCell.setAttribute('tabIndex', '0');
+export const updateFocus = (rowElements, cellCoord, shouldBlur = false) => {
```

Comment:
the fucntion is updateFocus, for me,` shouldFocus = true `is easier to understand.

Reply:
Hmm yeah I think I agree, I just wanted to avoid using the same name as `shouldRefocus`, but that is not the same as `shouldFocus`. So I'll change it.